impeller/tools/malioc.json

```diff
@@ -1699,16 +1699,16 @@
       "type": "Vertex",
       "variants": {
         "Position": {
-          "fp16_arithmetic": 100,
+          "fp16_arithmetic": 0,
           "has_stack_spilling": false,
           "performance": {
             "longest_path_bound_pipelines": [
               "load_store"
             ],
             "longest_path_cycles": [
-              0.0625,
```

Comment:
Here and below, most of these numbers appear to be going up. Is that a concern?

Reply:
These are vertex shaders, so generally its OK as most of our vertex shaders are incredibly fast. We do need to increase precision of input positions as these can overflow more easily.

Reply:
Additionally, these half float conversions only apply to iOS. gles and vulkan have been mostly using relaxed precision for many months